CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v1.0.4] 2020-01-27
+
+- Fixed hook deletion policy for sub chart that labels kube-system namespace.
```

Comment:
Please add the `Changed` / `Added` / `Deleted` format like below.

Reply:
Done.